crates/types/src/lib.rs

```diff
@@ -247,6 +248,10 @@ impl WasmRefType {
     pub fn is_gc_heap_type(&self) -> bool {
         self.heap_type.is_gc_heap_type()
     }
+
+    pub fn is_vmgcref_type(&self) -> bool {
+        self.heap_type.is_vmgcref_type()
+    }
```

Comment:
Do you think we can structure this as an enum or similar? E.g. a `.classify()` returning an enum of some kind. The differences between these functions feels really subtle and easy to get wrong, and getting it wrong feels like it could lead to unsafety pretty easily

Reply:
(We talked offline about this and decided to just rename the predicates).